docs/sample-app/photo-sharing/fragments/ios/getting_started.md

```diff
@@ -0,0 +1,44 @@
+PhotoSharing is a demonstration of building an iOS social networking App with [Amplify Libraries](~/lib/lib.md).
+
+## App features
+
+This is an App that you can perform:
+* Sign In/Sign Up/Sign Out
+* Post photo
+* Update your profile image 
+* Scroll to view a list of your created posts
+
+### Sign In/Sign Out
+
+The App allows you to sign up, sign in using [Amplify.Auth.signInWithWebUI](~/lib/auth/signin_web_ui.md) api.
+
+**Sign Up Flow:**
+![Sign Up](~/images/sample-app/photosharing/sign-up-flow.png)
+
+**Sign In Flow:**
+![Sign In](~/images/sample-app/photosharing/sign-in-flow.png)
+
+**Sign Out Flow:**
+![Sign Out](~/images/sample-app/photosharing/sign-out-flow.png)
+
+### Post Photo
+
+Once you are authenticated, you can create a post with selected image using [Amplify.DataStore.save](~/lib/datastore/data-access.md#create-and-update) and [Amplify.Storage.UploadData](~/lib/storage/upload.md) api
```

Comment:
```suggestion
Once the user has signed in, the app allows the user to create a new post using the plus icon located at the top right corner. Tapping on this will prompt the user to select from the photo library or take an photo with the camera. Once they have selected an image and entered a description for the post, the app will persist the Post using `Amplify.DataStore` and upload the image using `Amplify.Storage`.

> Recommendation: See [DataStore's Getting Started] guide and [Storage's Getting Started] guide for an alternative walk through of their respective categories.
```